boards/arm/frdm_k22f/board.cmake

```diff
@@ -1,15 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-set_ifndef(OPENSDA_FW daplink)
```

Comment:
Does it not make jlink as default runner? That would break usual workflow. And probably CI tests on hardware?

Reply:
No, pyocd should still be the default runner, because it's the first include below.

Reply:
yes, sorry overlooked that.